diff --git a/CMakeLists.txt b/CMakeLists.txt
index 8994119..9ccfacb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -113,8 +113,6 @@ IF(MSVC)
             ENDIF()
           ENDIF()
           STRING(REPLACE "/Zi" "/Z7" COMPILER_FLAGS ${COMPILER_FLAGS})
-          MESSAGE (STATUS "CMAKE_${COMPILER}_FLAGS_${BUILD_TYPE}= ${COMPILER_FLAGS}")
-          SET(CMAKE_${COMPILER}_FLAGS_${BUILD_TYPE} ${COMPILER_FLAGS})
         ENDIF()
       ENDFOREACH()
     ENDFOREACH()
diff --git a/libmariadb/ma_tls.c b/libmariadb/ma_tls.c
index b2de93e..042b11d 100644
--- a/libmariadb/ma_tls.c
+++ b/libmariadb/ma_tls.c
@@ -224,7 +224,7 @@ static my_bool ma_pvio_tls_compare_fp(MARIADB_TLS *ctls,
                                      const char *cert_fp,
                                      unsigned int cert_fp_len)
 {
-  const char fp[EVP_MAX_MD_SIZE];
+  char fp[EVP_MAX_MD_SIZE];
   unsigned int fp_len= EVP_MAX_MD_SIZE;
   unsigned int hash_type;
 
